store/actions/crypto.actions.tsx

```diff
@@ -20,31 +20,43 @@ export const readData = () => {
                 }
             })
             .catch(err => { console.error(err) })
-
     }
 }
 
 export const addCripto = (textInput: string) => {
     return async (dispatch: Dispatch<Action>, getState: () => RootState) => {
+
         const { cryptoList } = getState().cripto
+        const response = await fetch(`https://data.messari.io/api/v1/assets/${textInput.toLowerCase()}/metrics?fields=id,symbol,name,market_data/price_usd,market_data/percent_change_usd_last_24_hours`)
 
-        const newCrypto: Crypto | undefined = cryptos.find(({ name, initials }) => {
-            const addedCrypto: Crypto | undefined = cryptoList.find(({ name, initials }) =>
-                (textInput === name || textInput === initials)
-            )
-            return !addedCrypto && (name === textInput || initials === textInput)
-        })
-        if (newCrypto) {
-            dispatch({
-                type: ADD_CRYPTO,
-                payload: newCrypto
-            })
+        if (!response.ok) {
+
+            let message = "Hubo un problema"
+            const errorResponse = await response.json()
+            const errorID = errorResponse.status.error_message
+            if (errorID === "Not Found") message = "This cryptocurrency doesn't exist"
+            Alert.alert(message)
 
-            const updatedList = [...cryptoList]
-            try {
-                await AsyncStorage.setItem('@coinList', JSON.stringify(updatedList))
-            } catch (e) {
-                console.error(e)
+        } else {
+
+            const result = await response.json()
+
+            const addedCrypto: Crypto | undefined = cryptoList.find(({ name, symbol }) =>
+                (textInput.toLowerCase() === name.toLowerCase() || textInput.toLowerCase() === symbol.toLowerCase())
+            )
+            if (!addedCrypto) {
```

Comment:
[suggestion] Same as above, please try to work with affirmative conditionals whenever possible